session-helper/flatpak-session-helper.h

```diff
@@ -26,4 +26,10 @@
 #define FLATPAK_SESSION_HELPER_PATH_DEVELOPMENT "/org/freedesktop/Flatpak/Development"
 #define FLATPAK_SESSION_HELPER_INTERFACE_DEVELOPMENT "org.freedesktop.Flatpak.Development"
 
+typedef enum {
+  FLATPAK_HOST_COMMAND_FLAGS_CLEAR_ENV = 1 << 0,
+  FLATPAK_HOST_COMMAND_FLAGS_WATCH_BUS = 1 << 1,
+  FLATPAK_HOST_COMMAND_FLAGS_NONE = 0
```

Comment:
Perhaps `NONE` should be at the top? I know that's not consistent with `flatpak-portal.h` but it is consistent with `flatpak-installation.h` and makes more sense I think

Reply:
I've got into the habit of putting the `NONE` entry of enums at the bottom, as a trick to reduce conflicts and diffstat when updating them, while still being valid in C89. If we go from

```
typedef enum {
  FLAGS_FORCE = 1 << 0,
  FLAGS_VERBOSE = 1<< 1,
  FLAGS_NONE = 0
} Flags;
```

to

```
typedef enum {
  FLAGS_FORCE = 1 << 0,
  FLAGS_VERBOSE = 1<< 1,
  FLAGS_INTERACTIVE = 1 << 2,
  FLAGS_NONE = 0
} Flags;
```

then that's only one line of diff; but if we go from

```
typedef enum {
  FLAGS_NONE = 0,
  FLAGS_FORCE = 1 << 0,
  FLAGS_VERBOSE = 1<< 1
} Flags;
```

to

```
typedef enum {
  FLAGS_NONE = 0,
  FLAGS_FORCE = 1 << 0,
  FLAGS_VERBOSE = 1<< 1,
  FLAGS_INTERACTIVE = 1 << 2
} Flags;
```

then the `FLAGS_VERBOSE` line has also changed, because it's no longer the last one.

If we're allowing C99 syntax or if we're requiring gcc extensions (which we already do elsewhere for `g_autoptr` and friends), then we can avoid this by having a trailing comma; but neither of those would be considered OK in GLib, which is C89 because MSVC's C compiler is stuck in the 80s, and I'm trying not to diverge too far from coding styles that would fit into GLib.

Reply:
Ah ok